TowerForge/TowerForge/Commons/Protocols/Double+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Double+Extensions.swift
+//  TowerForge
+//
+//  Created by Rubesh on 14/4/24.
+//
+
+import Foundation
+
+extension Double {
```

Comment:
I think this file should be in the extensions folder

Reply:
Oh nice thanks for letting me know, that seems to be the issue. It's fixed now.